sandbox.html

```diff
@@ -148,6 +148,7 @@
       mapExtent,
       minScale,
       parts,
+      selectableLayers,
```

Comment:
The story description looks to me as if there should be an option to define _a single_ layers (not multiple).
> Within the selection dialog, only features from specified layer are selected if layer name was provided.
If no layer name was given, the selection takes place in current algorithm.

Are you sure that it should be multiple?

Proposal: Name the option just "layer" in case we want to identify a layer for other reasons in the future.

Reply:
The description was faulty, I updated description and added some bullet points on expected behavior when multiple layers are specified

Reply:
OK, then it's all good :+1: